app/views/studies/_sort_warning.html.erb

```diff
@@ -1 +1,5 @@
-<p class='text-danger sort-warning'><strong>Coordinate matrices must be sorted by gene indices (first column) in order to be parsed.</strong>  Please refer to our <a href='https://github.com/broadinstitute/single_cell_portal/wiki/Sorting-Matrix-Market-Coordinate-Matrices' target='_blank' style='text-decoration: underline;'>wiki</a> for more information.</p>
```

Comment:
This small change is a big deal!  

I added [a note](https://github.com/broadinstitute/single_cell_portal/wiki/Sorting-Matrix-Market-Coordinate-Matrices) to remove the linked wiki page after launching Ingest Pipeline.

Reply:
Yes - good catch.  I'm glad we wont have to force users to do that anymore.